modules_basic/scroller.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const h = require('../h')
+
```

Comment:
then maybe look at one other file that implements the pattern. I'll post next to a good example